### PR TITLE
Mobile skeleton

### PR DESCRIPTION
**Pull request by Eckybrazzz**
I created this PR to extend the existing mobile portfolio page. Added a lot of content. Added SVG Images. Created index.html and style.css based on the example that was available on the page figma.com. Tried to make an exact copy of the template on the sigma website.  The most important was to implement what I have learned this week.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,37 +37,278 @@ <h2>I'am a Software Developer</h2>
...
+                  <a href="#">Ruby</a>
+                </li>
+              </ul>
+            </nav>
+            <button type="button" class="professional-btn" onclick="window.location.href='#';">See Project</button>
+          </section>  
+          <!-- Start About me-->
+          <section class="about-me">
+            <div class="first">
+              <p>About me</p>
```
Comment (DanteAlonsoHT) [inline index.html:222]: - [ ] The "About me" title should be a header text, please use `h2`, for instance:

```
<h2>About me</h2>
```

File: index.html [hunk 2]
```diff
@@ -35,37 +37,278 @@ <h2>I'am a Software Developer</h2>
...
+                </li>
+                <li class="professional-btn1">
+                  <a href="#">bootstrap</a>
+                </li>
+                <li class="professional-btn1">
+                  <a href="#">Ruby</a>
+                </li>
+              </ul>
+            </nav>
+            <button type="button" class="professional-btn" onclick="window.location.href='#';">See Project</button>
```
Comment (DanteAlonsoHT) [inline index.html:197]: - [ ] For all your buttons, you should use the Figma design template to apply the styles (you can see them in the image below), please apply these styles using the next CSS:
```
button:hover {

}

button:active {

}
```

Button styles expected according to the Figma design you selected:

image

File: index.html [hunk 3]
```diff
@@ -35,37 +37,278 @@ <h2>I'am a Software Developer</h2>
...
+          </section>
+         </div>
+        </div>
+        </section>
+        <!-- My Recent Work-->
+        <section class="second">
+            <div class="second-page">
+              <div class="text">
+                <p>My Recent Works</p>
+              </div>
```
Comment (DanteAlonsoHT) [inline index.html:93]: "My recent works" should be a header text, please use `h2` to use semantic HTML. For instance:
```
<h2>My Recent Works</h2>
```

File: index.html [hunk 4]
```diff
@@ -35,37 +37,278 @@ <h2>I'am a Software Developer</h2>
...
+            <div class="text2">
+              <p>Multi-Post Stories</p>
+            </div>
+            <div class="text3">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </div>
+            <!-- Menu Items-->
+            <div class="language-container">
+              <nav class="nav-page2">
+                <ul class="link-tag">
```
Comment (DanteAlonsoHT) [inline index.html:108]: - [ ] Fix the extra space or add more space to make your design more similar to the Figma design you selected.
- [ ] I can not see the projects above the "Multi-Post Stories" project, kindly fix it.
- [ ] [OPTIONAL] Fix the font-weight in the button text according to the Figma design.

image

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @Eckybrazzz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Improve the description in your Pull Request, explain what you did, what is the main scope, which files do you changed or added, what you set up in your project in this pull request, and so on.

- [ ] Improve your README file, kindly you can check [This Example](https://github.com/DanteAlonsoHT/Leaderboard_API) to let you know what we expect from students 😉 

### 👇  _Check the comments under the review._ 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Eckybrazzz) [reply index.html:222]: Placed ```h2``` on line 22 and changed style.css for the correct size.

File: index.html (hunk 2 above)
Comment (Eckybrazzz) [reply index.html:197]: Fixed mouseover links with change in style.css

File: index.html (hunk 3 above)
Comment (Eckybrazzz) [reply index.html:93]: Placed ```h2``` on line 91 and changed style.css for the correct size.

File: index.html (hunk 4 above)
Comment (Eckybrazzz) [reply index.html]: All issues resolved

File: index.html (hunk 2 above)
Comment (Mihndim2020) [reply index.html:197]: - [ ] As  requested by the previous reviewer, as specified on the design, which is also part of the project requirements, please kindly add styles for the **Hover**, **Active**, **Disabled**,  states of your buttons.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @Eckybrazzz , 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there


 # Status: Require Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Please also consider:
- [OPTIONAL] fixing the alignment of your page, It is aligned to the left. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Leon-Mbegera — APPROVED** [review]
Hi @Eckybrazzz , 

# Approved 🎉 

- Well done on implementing all the requested changes by the previous reviewer 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it and proceed onto the next stage. :shipit: 
Congratulations! 🎉 

## Optional changes
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Eckybrazzz) [reply index.html]: fixed all Buttons and added style

**Pull request merged**